connect/src/main/java/com/stripe/android/connect/PayoutsView.kt

```diff
@@ -2,49 +2,38 @@ package com.stripe.android.connect
 
 import android.content.Context
 import android.util.AttributeSet
-import android.webkit.WebView
 import android.widget.FrameLayout
 import androidx.annotation.RestrictTo
-import com.stripe.android.connect.webview.StripeConnectWebViewClient
+import com.stripe.android.connect.webview.StripeConnectWebViewContainer
+import com.stripe.android.connect.webview.StripeConnectWebViewContainerImpl
 
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-class PayoutsView @JvmOverloads constructor(
+class PayoutsView private constructor(
     context: Context,
-    attrs: AttributeSet? = null,
-    defStyleAttr: Int = 0,
-    embeddedComponentManager: EmbeddedComponentManager? = null
-) : FrameLayout(context, attrs, defStyleAttr) {
+    attrs: AttributeSet?,
+    defStyleAttr: Int,
+    webViewContainerBehavior: StripeConnectWebViewContainerImpl,
+) : FrameLayout(context, attrs, defStyleAttr),
+    StripeConnectWebViewContainer by webViewContainerBehavior {
```

Comment:
👀 Composing the shared behavior here. Note the `private` constructor to hide the implementation from the user.